students/DennisLee/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,19 @@
+def print_fizz_buzz() -> None:
+    """
+    Print numbers 1 to 100, but 'Fizz' or 'Buzz' for multiples of 3 or 5
+
+    :return:  None.
+    """
+    for j in range(100):
+        i = j + 1
+        if i % 15 == 0:
+            i = 'FizzBuzz'
+        elif i % 3 == 0:
+            i = 'Fizz'
+        elif i % 5 == 0:
+            i = 'Buzz'
+        print(i)
+    return None
```

Comment:
Python functions return `None` by default, so there is no need to write `return None`